src/Features/Core/Portable/GenerateEqualsAndGetHashCodeFromMembers/GenerateEqualsAndGetHashCodeFromMembersCodeRefactoringProvider.cs

```diff
@@ -113,9 +113,13 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
             GetExistingMemberInfo(
                 containingType, out var hasEquals, out var hasGetHashCode);
 
+            var globalOptions = document.Project.Solution.Services.GetService<ILegacyGlobalOptionsWorkspaceService>();
```

Comment:
❓ Is it possible to leverage #66798 instead?

Reply:
Oh I don't know this service exists

Reply:
Yeah, I think it would be a good backup plan. But still, this code action depends on IPickMemberService, so it should also be checked

Reply:
I don't see it provides reading these values https://github.com/dotnet/roslyn/blob/1bc8f4913a88dacbef3f60e812e5763d5c9f837e/src/EditorFeatures/Core/Options/LegacyGlobalOptionsWorkspaceService.cs#L25